AWS-Transcription.py

```diff
@@ -1,86 +1,102 @@
-from __future__ import print_function
+#!usr/bin/env python3
+
 import boto3
 import time
-import os
```

Comment:
`os` isn't actually needed, I replaced it with `pathlib` as it's recommended for filepaths nowadays.